pkg/controllermanager/bootstrappers/bootstrap.go

```diff
@@ -7,23 +7,61 @@ package bootstrappers
 import (
 	"context"
 	"fmt"
+	"strconv"
 	"time"
 
 	"github.com/go-logr/logr"
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/discovery"
 	kubernetesclientset "k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 	"k8s.io/utils/clock"
+	"k8s.io/utils/ptr"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
+	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
 	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
+	gardenerutils "github.com/gardener/gardener/pkg/utils/gardener"
 	secretsutils "github.com/gardener/gardener/pkg/utils/secrets"
 	secretsmanager "github.com/gardener/gardener/pkg/utils/secrets/manager"
 	"github.com/gardener/gardener/pkg/utils/version"
 )
 
+const (
+	// ConfigMapsPerShootAnnotation is the annotation key for the amount of ConfigMaps created by Gardener in the project namespace.
+	ConfigMapsPerShootAnnotation = "gardener.cloud/configmaps-per-shoot"
+	// SecretsPerShootAnnotation is the annotation key for the amount of Secrets created by Gardener in the project namespace.
+	SecretsPerShootAnnotation = "gardener.cloud/secrets-per-shoot"
+	shootCountResource        = "count/shoots.core.gardener.cloud"
+)
```

Comment:
nit: empty line between the annotations and constant below, while glancing over it, I first assumed it's yet another annotation
```suggestion
const (
	// ConfigMapsPerShootAnnotation is the annotation key for the amount of ConfigMaps created by Gardener in the project namespace.
	ConfigMapsPerShootAnnotation = "gardener.cloud/configmaps-per-shoot"
	// SecretsPerShootAnnotation is the annotation key for the amount of Secrets created by Gardener in the project namespace.
	SecretsPerShootAnnotation = "gardener.cloud/secrets-per-shoot"
	
	shootCountResource        = "count/shoots.core.gardener.cloud"
)
```